File: client/src/main/java/com/github/diosa/dms/StartGUI.java
package com.github.diosa.dms;

import javafx.application.Application;
import javafx.fxml.FXMLLoader;
import javafx.scene.Parent;
import javafx.scene.Scene;
import javafx.stage.Stage;

import java.io.IOException;

public class StartGUI extends Application {
    @Override
    public void start(Stage stage) throws IOException {
        MainSceneController mainController = new MainSceneController();
        FXMLLoader loader = new FXMLLoader();
        loader.setLocation(StartGUI.class.getResource("/main.fxml"));
        Parent root = loader.load();
        stage.setScene(new Scene(root));
        stage.setTitle("Vehicle visualizer");

        loader.setController(mainController);
        stage.show();
    }
}

File: client/src/main/kotlin/com/github/diosa/dms/mainGUI/StartGUI.kt
package com.github.diosa.dms.mainGUI

import javafx.application.Application
import javafx.fxml.FXMLLoader
import javafx.scene.Parent
import javafx.scene.Scene
import javafx.stage.Stage
import java.io.IOException

class StartGUI : Application() {
    @Throws(IOException::class)
    override fun start(stage: Stage) {
        val loader = FXMLLoader()
        loader.location = StartGUI::class.java.getResource("/main.fxml")
        val root = loader.load<Parent>()
        stage.scene = Scene(root)
        stage.title = "Vehicle visualizer"
        loader.setController(MainSceneController())
//        mainController.collectionVisualise()
        stage.show()
    }
}